KieraAllen/viewing_party
# Registration page

Add skeleton for registration page

- Write simple welcome page test
- Pass initial welcome page test with route, controller, and view
- Write test for registration page skeleton
- Pass initial registration page test with route, controller, and view
- TESTING CO-AUTHORING …
- Re-testing co-authored commits

Create functionality for registering a new user and saving them to the database
- Create test for registering new user 
- Add registered and logged in message
- Add post route for registering new user
- Add create action in users controller
- Add name field to test and updated password confirmation label
- Create sad path test for if user already exists
- Test for sad path message for if user already exists
- Add check for user in database
- Migrate and seed

closes #14
